src/BridgeFinder.ts

```diff
@@ -169,14 +169,24 @@ export class BridgeFinder extends (EventEmitter as new () => TypedEmitter<Bridge
         }
 
         let bridgeID;
-        const hostname = await this.getHostnameFromIP(ipaddr);
-        logDebug('got hostname from IP:', hostname);
+        let hostname;
```

Comment:
Instead of a forward declaration, should the try block be split to narrow its scope?